File: libraries/core_network/src/main/java/com/tokopedia/core/shopinfo/models/shopmodel/ShopModel.java

package com.tokopedia.core.shopinfo.models.shopmodel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Get shop info object from tkpd shop
 */
@Deprecated
public class ShopModel {

    @SerializedName("info")
    @Expose
    public Info info;
    @SerializedName("stats")
    @Expose
    public Stats stats;

    public Info getInfo() {
        return info;
    }

}

File: features/merchant/product_detail/src/main/java/com/tokopedia/product/estimasiongkir/data/model/ShopModel.kt
package com.tokopedia.product.estimasiongkir.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ShopModel(
        @SerializedName("district_name")
        @Expose
        val districtName: String = ""
)